example/spectrogram/index.html

```diff
@@ -98,7 +98,26 @@ <h4>Options</h4>
                       <li><code>fftSamples</code> - number of FFT samples (<code>512</code> by default). Number of spectral lines and height of the spectrogram will be a half of this parameter.</li>
                       <li><code>frequenciesDataUrl</code> - URL to load spectral data from.</li>
                       <li><code>labels</code> - Whether or not to display frequency labels.</li>
+                      <li><code>colorMap</code> - Specifies the colormap to be used when rendering the spectrogram.</li>
```

Comment:
use 'a' instead of 'the' and use 'optional' somewhere

Reply:
I originally intended to and then noticed that the author did the opposite.  They specify "required" for all required parameters and don't add "optional" to the optional parameters.  I was trying to be consistent.  I could:
a) leave as is.
b) add optional to this one param.
c) add optional to all params.
Preference?

Reply:
it's fine like this then.